tensorrt_llm/_torch/pyexecutor/model_engine.py

```
@@ -703,52 +703,119 @@ def release_batch(result: ScheduledRequests | None):
+                    )
+
+                AutoTuner.get().print_profiling_cache()
+
+            if not (self._run_cuda_graphs
+                    or self._torch_compile_piecewise_cuda_graph):
+                return
+
+            logger.info(
+                f"Creating CUDA graph instances for {len(self._cuda_graph_batch_sizes)} batch sizes."
+            )
+            # Reverse the order of the cuda graph batch sizes to make smaller batch size graph could reuse larger batch size graph memory
```

Ruff (0.12.2)

Line too long (137 > 120)

(E501)